dash_docs/chapters/holoviews/index.py

```diff
@@ -316,8 +318,72 @@
     ),
     html.Div(examples['datashader-link-selections.py'][1], className="example-container"),
     rc.Markdown('''
+    ## Map Overlay
+    Most 2-dimensional HoloViews elements can be displayed on top of a map by
+    overlaying them on top of a `Tiles` element.  There are three approaches to 
+    configuring the map that is displayed by a `Tiles` element:
+    
+      1. Construct a `holoviews.Tiles` element with a WMTS tile server url.
+         E.g. `Tiles("https://maps.wikimedia.org/osm-intl/{Z}/{X}/{Y}@2x.png")`
+      2. Construct a `Tiles` element with a predefined tile server url using a
+         function from the `holoviews.element.tiles.tile_sources` module. E.g. `CartoDark()`
+      3. Construct a `holoviews.Tiles` element with an empty string, then use `.opts`
+         to supply a mapbox token and style.
+         E.g. `Tiles("").opts(mapboxstyle="light", accesstoken="pk...")`
+    
+    > **Coordinate Systems:**  Unlike the native plotly mapbox traces which accept
+      geographic positions in longitude and latitude coordinates, HoloViews expects
+      geographic positions to be supplied in Web Mercator coordinates 
+      (https://epsg.io/3857).  Rather than "longitude" and "latitude", horizontal and
+      vertical coordinates in Web Mercator are commonly called "easting" and "northing"
+      respectively. HoloViews provides `Tiles.lon_lat_to_easting_northing` and 
+      `Tiles.easting_northing_to_lon_lat` for converting between coordinate systems.
+    
+    This example displays a scatter plot of the `plotly.data.carshare` dataset on top
+    of the predefined `StamenTerrain` WMTS map.
+    '''),
+    rc.Markdown(
+        examples['mapbox-points.py'][0],
+        style=styles.code_container
+    ),
+    html.Div(examples['mapbox-points.py'][1],
+             className="example-container"),
+    rc.Markdown('''
+    ## Visualizing a Large Geographic Datasets with Datashader
```

Comment:
```suggestion
    ## Visualizing a Large Geographic Dataset with Datashader
```

I'm getting deja vu here! It's hard to see what this entire document is going to be like when rendered, when only reading through the git diffs, but this section seems like it's copying the title from the beginning of this document (line 258). Presumably one of them should be renamed?